django/core/checks/security/base.py

```diff
@@ -190,17 +190,22 @@ def check_ssl_redirect(app_configs, **kwargs):
 
 
 @register(Tags.security, deploy=True)
-def check_secret_key(app_configs, **kwargs):
+def check_secret_keys(app_configs, **kwargs):
+    def check_key(key):
+        return (
+            key and
+            len(set(key)) >= SECRET_KEY_MIN_UNIQUE_CHARACTERS and
+            len(key) >= SECRET_KEY_MIN_LENGTH and
+            not key.startswith(SECRET_KEY_INSECURE_PREFIX)
+        )
+
     try:
-        secret_key = settings.SECRET_KEY
+        secret_keys = settings.SECRET_KEYS
     except (ImproperlyConfigured, AttributeError):
         passed_check = False
     else:
-        passed_check = (
-            len(set(secret_key)) >= SECRET_KEY_MIN_UNIQUE_CHARACTERS and
-            len(secret_key) >= SECRET_KEY_MIN_LENGTH and
-            not secret_key.startswith(SECRET_KEY_INSECURE_PREFIX)
-        )
+        passed_check = all(check_key(key) for key in secret_keys)
+
```

Comment:
Is it worth checking that `secret_keys` != `isinstance(str)` (and raise a separate warning if it is)? It seems easy enough to accidentally do `SECRET_KEYS='a...b'` and whilst you'll still get the deployment warning (because each character won't pass, naturally) it won't _necessarily_ be obvious why, if your setting looks like it's otherwise a 'good' candidate. Moreso if you consider that suffixing your accidental string usage, with a comma would fix it (turning it into a tuple) or vice versa.

Reply:
Good idea!

Reply:
I think adding `SECRET_KEYS` to `tuple_settings` covers this. See e5300963294a62b7321aa7e99f802e4d8557dd80